Applications/WebPortal/WebApp/library/source/components/country_selection_box.tsx

```diff
@@ -134,4 +127,5 @@ export class CountrySelectionBox extends React.Component<Properties, State> {
       }
     }
   });
+  private options: any[];
```

Comment:
This should be of type ```JSX.Element[]```, if you can narrow it down even further, do so. You might be able to resolve down to something like ```JSX.OptionElement```, not too sure but try to find the narrowest type.